config/config.sample.yaml

```
@@ -38,6 +38,10 @@ bridge:
   disableInviteNotifications: false
   # Disable Room Topic echos from matrix
   disableRoomTopicNotifications: false
+  # Enable reactions from matrix to discord. This will only respond to
```

Capitalise names to differentiate them from the regular English words.

```suggestion
  # Enable reactions from Matrix to Discord. This will only respond to
```